### DISCUSSION TITLE
Does esp32 support BLE Mesh

### DISCUSSION THREAD

**Discussion by git4Chub** [zephyrproject-rtos/zephyr#49078] [Q&A]
I 've tried on BLE scan/adv/conn features on esp32, However，when I enabled Mesh config, and tried Mesh provisoning, my esp32 crashed, so I want to know if mesh is supported on esp32. Thx!

**Comment by rftafas** [comment]
AFAIK, it was not thoroughly tested so current state is of no support. Since this is not on our major roadmap, any debug info could enable us to help if you are willing to get hands on.
